src/main/resources/application.yaml

```diff
@@ -175,9 +175,9 @@ ccdGatewayUrl: ${CCD_GW_URL:http://localhost:3453}
 
 docmosis.accessKey: ${DOCMOSIS_ACCESS_KEY}
 docmosis.endpoint: ${DOCMOSIS_ENDPOINT}
-docmosis.render.uri: /rs/render
-docmosis.status.uri: /rs/status
-docmosis.convert.uri: /rs/convert
+docmosis.render.uri: ${DOCMOSIS_RENDER_URL:/rs/render}
+docmosis.status.uri: ${DOCMOSIS_STATUS_URL:/rs/status}
+docmosis.convert.uri: ${DOCMOSIS_CONVERT_URL:/rs/convert}
 
```

Comment:
These were wrongly committed in one of the previous PRs and are now being realigned with what's on master.

Reply:
Thanks